### ISSUE TITLE
eix does not consider profiles in other repositories.

### ISSUE THREAD

**Issue by dElogics** [vaeth/eix#121]
One of my custom repositories inherits from one of the gentoo repository's profiles and adds some custom masks, USE flags etc...
It seems (at least for package.mask) eix does no read this custom profile.

repos.conf/custom-profiles.conf:
[custom-profiles]
location = /etc/custom-profiles-gentoo
sync-type =

custom-profiles-gentoo # find -type f
./profiles/repo_name
./profiles/profiles.desc
./profiles/client/parent
./profiles/client/repo_name
./profiles/client/package.mask
./metadata/layout.conf

cat ./profiles/client/package.mask
<sys-kernel/gentoo-sources-6.1.70
>sys-kernel/gentoo-sources-6.1.70

emerge respects this -- 
emerge -pv --getbinpkg --oneshot sys-kernel/gentoo-sources

!!! All ebuilds that could satisfy "sys-kernel/gentoo-sources" have been masked.
!!! One of the following masked packages is required to complete your request:
- sys-kernel/gentoo-sources-6.10.3::gentoo (masked by: package.mask, ~amd64 keyword)
/etc/custom-profiles-gentoo/profiles/client/package.mask:
....

However eix does not show sys-kernel/gentoo-sources as masked.

**Comment by vaeth** [comment]
package.mask in overlays take only effect for the packages in that overlay: eix goes a long way to make sure to copy that portage behavior. You can verify that behavior by creating a sys-kernel/gentoo-sources package in your profile, and you will see that eix does mask it.

I also verified that adding sys-kernel/gentoo-sources to package mask of the mv overlay. does not prevent me to emerge gentoo-sources from the gentoo repository.

The masking probably works for you in portage by accident because - as I assume - you use some additional profile trickery in portage.

**Comment by dElogics** [comment]
The repository inherits from the gentoo (default) repository -- 
layout.conf 
repo-name = custom-profiles
masters = gentoo
profile-formats = portage-2

That's why it works. eix must also respect this behavior.

All of the profile's parent profiles are within this repository itself or inherit from one of the stock gentoo profile.

**Comment by vaeth** [comment]
> That's why it works.

No. The mv repository with which I tried also has "masters = gentoo".
